.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I assume that this file is not necessary, @BraneOpacic21?

Reply:
I would agree, considering it only has an empty object literal inside 😸

Reply:
@bulic please remove it You can place it in .gitignore file.